content/docs/command-reference/exp/run.md

```diff
@@ -170,3 +170,91 @@ CPU cores).
   regardless of this flag.
 
 - `-v`, `--verbose` - displays detailed tracing information.
+
+## Examples
+
+> These examples are based on our
+> [Get Started](/doc/tutorials/get-started/experiments), where you can find the
+> actual source code.
+
+<details>
+
+### Expand to prepare the example ML project
+
+Clone the DVC repo and download the data it <abbr>depends</abbr> on:
+
+```dvc
+$ git clone git@github.com:iterative/example-get-started.git
+$ cd example-get-started
+$ dvc pull
+```
+
+Let's also install the Python requirements:
+
+> We **strongly** recommend creating a
+> [virtual environment](https://python.readthedocs.io/en/stable/library/venv.html)
+> first.
+
+```dvc
+$ pip install -r src/requirements.txt
+```
+
+</details>
+
+Let's check the latest metrics of the project:
+
+```dvc
+$ dvc metrics show
+Path         avg_prec    roc_auc
+scores.json  0.60405     0.9608
+```
+
+For this experiment, we want to see the results for a smaller dataset input, so
+let's limit the data to 20 MB and reproduce the pipeline with `dvc exp run`:
+
+```dvc
+$ truncate --size=20M data/data.xml
+$ dvc exp run
+...
+Reproduced experiment(s): exp-44136
+Experiment results have been applied to your workspace.
+
+$ dvc metrics diff
+Path         Metric    Old      New      Change
+scores.json  avg_prec  0.60405  0.56103  -0.04302
+scores.json  roc_auc   0.9608   0.94003  -0.02077
+```
+
+The results in the `exp-44136` experiment seem to be worst (expected), as the
+`dvc metrics` commands show.
```

Comment:
```suggestion
The `dvc metrics` command shows the difference in performance in the `exp-44136` experiment.
```

I'd suggest leaving out any commentary here on expectations since it may not be obvious and might require too much knowledge of the example problem for people who haven't gone through the get started.

Reply:
Good call, will apply.